spec/std/enum_spec.cr

```diff
@@ -37,6 +37,14 @@ describe Enum do
     end
   end
 
+  it "creates an enum instance from a macro symbol" do
+    enum_value =
+      {% begin %}
+        SpecEnum.from_self({{ :two }})
+      {% end %}
```

Comment:
Note: I'm not sure why I need to use macro begin/end here, otherwise it fails with:
`no overload matches 'SpecEnum.from_self' with type Symbol`
I guess it's because auto-casted symbols are resolved earlier and not when the method is actually called?